ci-operator/step-registry/acm/interop-p2p/cluster-install-vsphere/acm-interop-p2p-cluster-install-vsphere-commands.sh

```
@@ -0,0 +1,644 @@
+    # --- vSphere CA cert secret ---
+    : "Creating vsphere-certs secret"
+    if [[ -n "${vcenterCert}" ]]; then
+        oc -n "${clusterName}" create secret generic vsphere-certs \
+            --from-literal=cacert="${vcenterCert}" \
+            --dry-run=client -o yaml --save-config | oc apply -f -
+    else
+        # Empty secret — Hive will attempt to verify against system trust.
+        oc create -f - --dry-run=client -o json --save-config |
+        jq -c --arg ns "${clusterName}" \
+            '.metadata.namespace = $ns | .metadata.name = "vsphere-certs"' |
+        oc apply -f - 0<<'ocEOF'
```

Shellcheck (0.11.0)

[error] This redirection overrides piped input. To use both, merge or pass filenames.

(SC2259)